Networking/Networking/Remote/FeatureFlagRemote.swift

```diff
@@ -28,16 +28,13 @@ public class FeatureFlagRemote: Remote, FeatureFlagRemoteProtocol {
 
 public enum RemoteFeatureFlag: Decodable {
     case storeCreationCompleteNotification
-    case oneDayAfterStoreCreationNameWithoutFreeTrial
```

Comment:
We might want to add a to-do list item for removing this flag in the remote some time in the future.

Reply:
Good point. Added the following to the parent issue #10374 as a subtask.

> Remove woo_notification_nudge_free_trial_after_1d remote feature flag once both iOS and Android release this new profiler flow.